src/aws/lambda.tf

```diff
@@ -82,8 +82,8 @@ resource "aws_lambda_function" "marsha_update_state_lambda" {
   function_name    = "${terraform.workspace}-marsha-update-state"
   handler          = "index.handler"
   runtime          = "nodejs8.10"
-  filename         = "dist/marsha_update_state.zip"
-  source_code_hash = "${base64sha256(file("dist/marsha_update_state.zip"))}"
+  filename         = "dist/marsha_update-state.zip"
+  source_code_hash = "${base64sha256(file("dist/marsha_update-state.zip"))}"
```

Comment:
I think the same problem exists with the 2 other lambdas: "lambda-configure" and "lambda-encode".

Reply:
no the name is the good one for configure and encode

Reply:
ok sorry about that!